### ISSUE TITLE
pkg:nix job is too verbose

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#9441]
There is a failure at https://github.com/coq/coq/pull/8930

https://gitlab.com/coq/coq/-/jobs/153288596

Job's log exceeded limit of 4194304 bytes.

**Comment by ejgallego** [comment]
This is likely a side effect of some gilab changes we are doing and should be closed cc: @Zimmi48 

You see so much output because Nix is not using the cache.

**Comment by Zimmi48** [comment]
The raw full logs say:

```
warning: unable to download 'https://cachix.cachix.org/nix-cache-info': Couldn't connect to server (7)
```

which should be unrelated to the other changes we are doing, but should be fixed by #9440 nonetheless.
